src/sage/rings/function_field/drinfeld_modules/finite_drinfeld_module.py

```diff
@@ -663,14 +678,13 @@ def is_isogenous(self, psi):
         - they have the same rank
         - the characteristic polynomial of the Frobenius endomorphism
           for both Drinfeld modules are equal.
-
         """
         if not isinstance(psi, DrinfeldModule):
             raise TypeError("Input must be a Drinfeld module")
         if self.category() != psi.category():
             raise TypeError("Drinfeld modules are not in the same category")
-        return self.rank() == psi.rank() and\
-                self.frobenius_charpoly() == psi.frobenius_charpoly()
+        return self.rank() == psi.rank() and \
+               self.frobenius_charpoly() == psi.frobenius_charpoly()
```

Comment:
After all it looks better with the `and` at the begining of the line:

```suggestion
        return self.rank() == psi.rank() \
               and self.frobenius_charpoly() == psi.frobenius_charpoly()
```